marklogic-client-api/src/test/java/com/marklogic/client/test/dataservices/MultipleValuesAsArrayTest.java

```diff
@@ -0,0 +1,137 @@
+ */
+package com.marklogic.client.test.dataservices;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.dataservices.CallBatcher;
+import com.marklogic.client.dataservices.CallManager;
+import com.marklogic.client.document.JSONDocumentManager;
+import com.marklogic.client.impl.NodeConverter;
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.client.io.ReaderHandle;
+import com.marklogic.client.io.marker.BufferableHandle;
+import com.marklogic.client.query.DeleteQueryDefinition;
+import com.marklogic.client.query.QueryManager;
+import com.marklogic.client.test.Common;
+
+public class MultipleValuesAsArrayTest {
+
+    private final static String ENDPOINT_DIRECTORY = "/javaApi/test/multipleValuesAsArrayTest/";
+
+    private static DatabaseClient db = Common.connect();
+    private static CallManager callMgr = CallManager.on(db);
+    private static EndpointUtil endpointUtil = new EndpointUtil(callMgr, ENDPOINT_DIRECTORY);
+    private static CallManager.CallableEndpoint callableEndpoint;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @BeforeClass
+    public static void setup() {
+        DatabaseClient adminClient = Common.newServerAdminClient("java-unittest-modules");
+        JSONDocumentManager docMgr = adminClient.newJSONDocumentManager();
+
+        DocumentMetadataHandle docMeta = new DocumentMetadataHandle();
+
+        docMeta.getPermissions().add("rest-reader", DocumentMetadataHandle.Capability.EXECUTE);
+        endpointUtil.setupEndpointMultipleRequired(docMgr, docMeta, "textDocument");
+
+        adminClient.release();
+    }
+
+    @AfterClass
+    public static void release() {
+        DatabaseClient adminClient = Common.newServerAdminClient("java-unittest-modules");
+
+        QueryManager queryMgr = adminClient.newQueryManager();
+        DeleteQueryDefinition deletedef = queryMgr.newDeleteDefinition();
+        deletedef.setDirectory(ENDPOINT_DIRECTORY);
+        queryMgr.delete(deletedef);
+
+        adminClient.release();
+    }
+
+    @Test
+    public void copyToBytesHandleTest() {
+        BufferableHandle handle = null;
+        assertNull(NodeConverter.copyToBytesHandle(handle));
+
+        BufferableHandle[] handleArray = null;
+        assertNull(NodeConverter.copyToBytesHandle(handleArray));
+
+        handleArray = new BufferableHandle[1];
+        handleArray[0] = handle;
+        assertNotNull(NodeConverter.copyToBytesHandle(handleArray));
+        assertNull(NodeConverter.copyToBytesHandle(handleArray)[0]);
```

Comment:
Good to have this test of the utility method.

Would it be useful to add a test that a non-empty BufferableHandle is copied?  A StringHandle is an easy BufferableHandle to work with.